packages/axelar-wasm-std/src/address.rs

```diff
@@ -19,6 +20,7 @@ pub enum AddressFormat {
     Eip55,
     Sui,
     Stellar,
+    Aleo,
```

Comment:
I don't think we need this change in the main repo. The address format is only used by the voting verifier. The aleo voting verifier should be maintained on your own fork. The message id is the only change we need in the universal repo.